tests/chainer_tests/datasets_tests/test_pickle_dataset.py

```diff
@@ -1,11 +1,43 @@
+import ctypes
 import io
+import multiprocessing
 import os
 import sys
 import unittest
 
+import mock
+
 from chainer import datasets
 from chainer import testing
 from chainer import utils
+from chainer.datasets.pickle_dataset import _FileReader
```

Comment:
Please avoid importing a class, as per the OpenStack coding guideline.

Let's import this way: `from chainer.datasets import pickle_dataset`

Reply:
OK, I fixed it.